### PR TITLE
Minor cargo supplies tweak

### PR DESCRIPTION
**Pull request by corp-0**
# Minor cargo supplies tweak

### Purpose
Changed the content of a couple crates and added a new one.

### Notes:
CHANGED: Steak crate now contains 5 steaks (previously 2) and a pepper shaker.  The boys need to eat and hopefully this will encourage people to don't become cannibals so quickly. No changes on price.

CHANGED: Cleaning crate now contains a trashbag, space cleaner, a mop and a bucket (previously it was lacking the trashbag and bucket). No changes on price.

ADDED: New EVA crate on Emergency category. Contains an entire mining hardsuit (because we don't have other hardsuits but Synd one), o2 tank and breath mask. The reasoning is having a particular crate for "all porpuses" hardsuit without the mining specific things (tcoms, jumpsuit). This could eventually have another hardsuit object with a neutral sprite. Price is same as shaft miner uniform crate.

REMOVED: Metal. glass and plasteel crates from the Engineering category.

ADDED: Metal, glass and plasteel crates under a new Materials category. Added solid plasma crate with 5 plasma at 5000 credits under the same category.

ADDED: Science category. Currently only order avaible is the chemist kit crate with 3 generic containers, a bucket and chemist labcoat.

ADDED: requested by FishFish, a new Box with glasses.

FIXED: a missing reference on janitor toolbet. I assumed it was the trashbag.

### Please make sure you have followed the self checks below before submitting a PR:

- Code is sufficiently commented
- Code is indented with tabs and not spaces
- The PR does not include any unnecessary .meta, .prefab or <b>.unity (scene) changes</b>
- The PR does not bring up any new compile errors
- The PR has been tested in editor
- Any new files are named using PascalCase (to avoid issues on case sensitive file systems)
- Any new / changed components follow the [Component Development Checklist](https://github.com/unitystation/unitystation/wiki/Component-Development-Checklist)
- Any new objects / items follow the [Creating Items and Objects Guide](https://github.com/unitystation/unitystation/wiki/Creating-Items-and-Objects%3A-Now-With-Prefab-Variants) (especially concerning the use of prefab variants)
- The PR has been tested in multiplayer (with 2 clients and late joiners, if applicable)
- The PR has been tested with round restarts.
- The PR has been tested on moving / rotating / rotated-before-joining matrices (if applicable)

### CODE REVIEW

**Comment by fomalsd** [comment]
looking good, ping when you’re ready to get this merged

**Comment by corp-0** [comment]
@fomalsd all done!

File: UnityProject/Assets/Scripts/UI/Cargo/CargoData.asset [hunk 1]
```diff
@@ -91,335 +96,18 @@ MonoBehaviour:
...
-      - {fileID: 3011710637427380133, guid: 5a4f48d58142749de92013eb1e1d0523, type: 3}
-      - {fileID: 3011710637427380133, guid: 5a4f48d58142749de92013eb1e1d0523, type: 3}
-      - {fileID: 3011710637427380133, guid: 5a4f48d58142749de92013eb1e1d0523, type: 3}
-      - {fileID: 7106618204054676533, guid: aa1e5a44ff182415dac90bed74ef8bfa, type: 3}
-      - {fileID: 7106618204054676533, guid: aa1e5a44ff182415dac90bed74ef8bfa, type: 3}
-      - {fileID: 7106618204054676533, guid: aa1e5a44ff182415dac90bed74ef8bfa, type: 3}
-      - {fileID: 835768756810601639, guid: 3d08b6dbf08774e8681378945dec4edf, type: 3}
-      - {fileID: 835768756810601639, guid: 3d08b6dbf08774e8681378945dec4edf, type: 3}
-      - {fileID: 835768756810601639, guid: 3d08b6dbf08774e8681378945dec4edf, type: 3}
-    - OrderName: Cook Set
```
Comment (fomalsd) [inline UnityProject/Assets/Scripts/UI/Cargo/CargoData.asset]: wait, why did you remove a whole bunch of sets? assistant, cook, bartender...

**Review by fomalsd — CHANGES_REQUESTED** [review]
a whole bunch of job-related crates are now gone

File: UnityProject/Assets/Scripts/UI/Cargo/CargoData.asset (hunk 1 above)
Comment (fomalsd) [reply UnityProject/Assets/Scripts/UI/Cargo/CargoData.asset]: you added a drinking glass but wiped all the uniform sets it seems

File: UnityProject/Assets/Scripts/UI/Cargo/CargoData.asset [hunk 2]
```diff
@@ -95,7 +95,18 @@ MonoBehaviour:
...
+    - OrderName: Chemist kit Crate
+      CreditsCost: 800
+      Crate: {fileID: 7880601349656009979, guid: 907d16a099bbf496d9fc11b8c260ccf9,
+        type: 3}
+      Items:
+      - {fileID: 7785875528613525472, guid: f866b44bcdd7d734885f7c49999d85c9, type: 3}
+      - {fileID: 7785875528613525472, guid: f866b44bcdd7d734885f7c49999d85c9, type: 3}
+      - {fileID: 7374481989050852271, guid: 1fada2a01fe5e49208acb17e93095002, type: 3}
+      - {fileID: 7516689663430431150, guid: 8aa459a6447baed4a95b189f4950f59f, type: 3}
+      - CategoryName: Uniform Sets
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/UI/Cargo/CargoData.asset]: here was the problem. Fucked up the indentation, unity freaked and purged the entire category.

File: UnityProject/Assets/Scripts/UI/Cargo/CargoData.asset (hunk 1 above)
Comment (corp-0) [reply UnityProject/Assets/Scripts/UI/Cargo/CargoData.asset]: fixed and this time double checked playing on editor.

File: UnityProject/Assets/Scripts/UI/Cargo/CargoData.asset (hunk 2 above)
Comment (fomalsd) [reply UnityProject/Assets/Scripts/UI/Cargo/CargoData.asset]: did you actually modify asset file manually? they are only intended to be edited from editor, keep that in mind
